docs/src/integrate/dxe_core.md

```diff
@@ -493,26 +493,22 @@ For detailed memory allocation behavior, see [DXE Core Memory Management](../dxe
 
 ### 9.3 Resource Descriptor HOB Version Support
 
-Patina DXE Core supports two mutually exclusive formats for Resource Descriptor HOBs: V1 (legacy) and V2 (modern,
-with cache attributes). The version supported is selected at compile time using a Cargo feature flag:
+Patina DXE Core supports two mutually exclusive formats for Resource Descriptor HOBs: v1 (what EDK II uses) and v2
+(cache attribute information). The version supported is selected at compile time using a Cargo feature flag:
```

Comment:
Would it be more apt to say this is v1 + additional cache attribute information?